src/pages/contributors/james-nord.adoc

```diff
@@ -0,0 +1,146 @@
+= James Nord
+:page-name: James Nord
+:page-linkedin: jtnord
+:page-twitter: 
+:page-github: jtnord
+:page-email:
+:page-image: avatar/james-nord.jpeg
+:page-pronouns:
+:page-location: Hampshire, United Kingdom
+:page-firstcommit: 2009
+:page-datepublished: 2024-12-10
+:page-featured: true
+:page-intro: James Nord is a software engineer based out of the UK that has been part of the Jenkins community since Kohsuke Kawaguchi first announced the move to open source. James started out with more basic usage of the Jenkins platform, but he soon started writing plugins to solve problems and fill any gaps that he found. Attributing his success with Jenkins to the ecosystem and extensibility it offers, James eventually collaborated directly with the project. Documenting bugs and starting his contributions in the early days of mailing lists was just the start of a much longer journey. He was one of Kohsuke's first customers, and then joined CloudBees as an employee. Since then, James has helped push Jenkins further along and been a driving force for necessary innovation, such as supporting FIPS 140 and making Jenkins more secure for government work. Throughout his involvement with Jenkins, James has enjoyed spreading his focus to different areas of the project, making sure to keep his work interesting and varied.
+
+== What is your background prior to contributing to Jenkins?
+
+After receiving my master's in electrical and electronic engineering from Manchester University, I moved to Sweden and started life as a research engineer at a university.
+I was working with location-aware applications and mass distribution of media on the internet.
+This was all before mobile phones had built-in GPS systems.
+After some time in Sweden, I moved to Hampshire in the UK to work on the "head end" (backend) for a company catering to satellite and cable TV operators.
+Within broadcasting, they call it "head end" and it's everything that drives automation of the broadcast.
+This included EPGs, scheduling, and encryption of data and devices as well.
+So I was working on the management of life cycles, encryption for devices, and other various necessities.
+The company was acquired by Cisco partway through my ten-year tenure there, which is when I got my first exposure to Hudson.
```

Comment:
something snot quite right here.
I think this implies I got the exposure at Cisco, but it was with the company before it was acquired that I started using it.

Reply:
Looks good, thanks